docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,44 @@
+# AntennaAttributionDocumentReporter
+The AntennaAttributionDocumentReporter generates an attribution document in .pdf format by leveraging the [Eclipse 
+Antenna][1] project. It generates a blank PDF file without any configuration, but the attribution document can be 
+customized by providing a template JAR, or a directory with template PDF files. 
+
+There is a guide in the [Antenna project documentation][2], which describes how such a template JAR file for the 
+customization has to be created. This has to be done before using the reporter. 
+
+In addition to the JAR file technique, there is an alternative way to customize the attribution document. For this, a 
+directory with template PDF files has to be provided. The PDF file names have to match the following:
+ * cover.pdf: Template PDF file for the cover page.
+ * copyright.pdf: Template PDF file for the project copyright page.
+ * content.pdf: Template PDF file for the packages of the project.
+ * back.pdf: Template PDF file for the back page.
+
+## Report options
+* `template.id`: Specifies the unique ID of the provided template JAR. The ID has to be specified in the template JAR 
+implementation. This option has  is only necessary if the reporter is used with the provided template JAR. 
+* `template.path`: This option has to point to a template JAR file, or a directory with the template PDF files.
+
+## Command Line
+To use the reporter with a template JAR file, pass it to the `--report-option` option of the _report_: 
+
+```bash
+cli/build/install/ort/bin/ort report
+  -i [scanner-output-path]/scanner-result.yml
+  -o [reporter-output-path]
+  --report-formats AntennaAttributionDocument
+  --report-option AntennaAttributionDocument=template.id=[template-id]
+  --report-option AntennaAttributionDocument=template.path=[template-jar-path]/custom-template.jar
+```
+
+To use the reporter with PDF files stored in a directory, pass it to the `--report-option` option of the _report_:
```

Comment:
"To use the _AntennaAttributionDocument_ report format with template PDF files, pass the directory the PDF files are located in as the `template.path` via the `--report-option` (or `-O`) option to the _report_ command:"

Reply:
Done.